projects/cdk/utils/color/rgba-to-hex.ts

```diff
@@ -3,11 +3,17 @@ export function tuiRgbaToHex(color: string): string {
         throw new Error('Invalid RGBa');
     }
 
-    const rgb: number[] =
+    const rgb =
         (color
             .replaceAll(/\s/g, '')
-            .match(/^rgba?\((\d+),(\d+),(\d+),?([^,\s)]+)?/i) as unknown as number[]) ??
-        [];
+            .match(/^rgba?\((\d+),(\d+),(\d+),?([^,\s)]+)?/i) as unknown as [
```

Comment:
Are we sure there's exactly this many items in the array here? Maybe it's better to assume some of them might not be there?

Reply:
fixed